### ISSUE TITLE
unify the shell scripts that print random ingredients into one awk script

### ISSUE THREAD

**Issue by vielmetti** [vielmetti/menu-planner#9]
I now have `ingredients.csv` all in one place, so make a new script `ingredients.awk` that runs like so

```
% awk -f ingredients.awk cool-and-crunchy 
fennel
```

to produce one example, or just

```
% awk -f ingredients.awk
fennel
spinach
nuts
cauliflower
polenta
coriander
```

to produce one example from each category

Try the best I can to avoid calling `gshuf`, instead use `srand`,`rand` and load all the data into arrays from which to pick.

**Comment by vielmetti** [comment]
some useful code at https://stackoverflow.com/questions/7514896/select-random-3000-lines-from-a-file-with-awk-codes

**Comment by vielmetti** [comment]
This would be easier in a language with first-class array objects, but I console myself that I'm still at less than 250 milliseconds to do it all even with exec'ing shuf a bunch of times and munging strings around, so that's fine.

**Comment by vielmetti** [comment]
Fixed in 5129fb5 I think

**Comment by vielmetti** [comment]
Done enough for now, there's some TODOs for parsing the command line that aren't necessary right now but that would be useful in some future version of this program that exposes a query and selection interface on `ingredients.csv`.
